class.h

```diff
@@ -0,0 +1,118 @@
+# include <iostream>
+# include <string>
+# include <vector>
+
+using namespace std;
+
+
+class Musician
+{
+
+private:
+	string name;
+	int fee;
+	int age;
+	string country;
+	bool band;
+
+protected:
+public:
+	Musician() {}
+	Musician(string name, int fee, int age, string country, bool band)
+	{
+		this->name = name;
+		this->fee =fee;
+		this->age = age;
+		this->country = country;
+		this->band = band;
+	}
+	~Musician(){}
+	string getName()
+	{ 
+		return name;
+	}
+	int getFee()
+	{
+		return fee;
+	}
+	int getAge()
+	{
+		return 0;
+	}
+	string getCountry()
+	{
+		return country;
+	}
+	bool getBand() 
+	{
+		return band;
+	}
+};
```

Comment:
Please separate methods and their budies by creating another .cpp file, where bodies of methods will be